rotkehlchen/api/v1/schemas.py

```diff
@@ -705,11 +704,25 @@ def make_history_event_filter(
         else:
             filter_query = HistoryEventFilterQuery.make(**common_arguments)
 
-        return {
+        return self.validation_proxied_information(data) | {
             'filter_query': filter_query,
-            'group_by_event_ids': data['group_by_event_ids'],
         }
 
+    def make_extra_filtering_arguments(self, data: dict[str, Any]) -> dict[str, Any]:
+        """
+        generates the extra fields to be included in the filter_query dictionary
+        """
+        return {
+            'limit': data['limit'],
+            'offset': data['offset'],
+        }
+
+    def validation_proxied_information(self, data: dict[str, Any]) -> dict[str, Any]:
```

Comment:
This name makes more sense. Could not understand the previous one

```suggestion
    def genrate_fields_post_validation(self, data: dict[str, Any]) -> dict[str, Any]:
```